Marlin/src/lcd/menu/menu_bed_tramming.cpp

```diff
@@ -235,7 +228,7 @@ static void _lcd_goto_next_corner() {
   }
 
   bool _lcd_bed_tramming_probe(const bool verify=false) {
-    if (verify) line_to_z(BED_TRAMMING_Z_HOP); // do clearance if needed
+    if (verify) line_to_z(current_position.z + (BED_TRAMMING_Z_HOP)); // do clearance if needed
```

Comment:
Code like this will need to be tested since it may now need special handling for the first point. We don't want to just raise over the current height before we've moved down to the bed.

Reply:
Upon initial review this change seems to be safe and won't adversely affect behavior.